# Methods

## Map construction and availability  

Causal interactions relevant to AD and PsO were extracted from biomedical literature and represented in a diagrammatic visualisation, referred to as the map, using the [CellDesigner tool](https://www.celldesigner.org) and following the Systems Biology Graphical Notation (SBGN) standard (see [SBGN learning page](https://sbgn.github.io/learning) for details). The only exception was in the representation of drugs, where we utilized the CellDesigner-specific glyph. The ISD map was subsequently uploaded to the [MINERVA platform](https://minerva.pages.uni.lu/doc/) for straightforward access and exploration. Further details are provided below.

## Capturing AD- and PsO-related causal interactions from literature  

The ISD map was developed by adhering to many recommendations from the recently published [guidelines for constructing disease maps](https://doi.org/10.3389/fbinf.2023.1197310). Initially, we manually curated reviews and original peer-reviewed research articles suggested by domain experts from the BIOMAP consortium. While most of these expert-selected papers focused on experimentally validated causal interactions specific to AD and PsO, some described experimentally validated immune processes in general, which were included to fill gaps in disease-specific information. Most causal interactions were assigned to compartments (such as organs, tissues, cells, and organelles) based on information provided in the papers; in certain cases, this assignment was inferred using common knowledge about the usual locations where these interactions occur. We also documented details related to causal interactions (e.g., gene variants, phosphorylated or citrullinated residues in proteins) when available in the literature.

## Encoding causal interactions into diagrams
The identified causal interactions were encoded into a diagrammatic visualization format using the  [CellDesigner tool](https://www.celldesigner.org), following its standard visual syntax, the CellDesigner Systems Biology Markup Language (SBML) extension. This encoding process adhered to the Systems Biology Graphical Notation (SBGN) standard (see [SBGN learning page](https://sbgn.github.io/learning) for details). We utilized both the SBGN Process Description (PD) and Activity Flow (AF) languages. In PD, a causal interaction is depicted as a state transition of a biochemical entity (the regulated entity), including biochemical details about the type of molecular transition and how it is influenced by a regulator entity. In AF, a causal interaction is shown as a simple directed link between two entities, indicating the type of regulation, such as activation or inhibition

## Identification and annotation of the map elements
To ensure compatibility with systems biology tools and external databases, all entities (proteins, RNAs, genes, complexes, metabolites, drugs, and phenotypes), compartments (organelles, cells, tissues, and organs), and interactions in the map were thoroughly annotated. This process followed the [Minimal Information Requested in the Annotation of Models (MIRIAM) guidelines](https://doi.org/10.1038/nbt1156), a standard for annotating and curating computational models and maps. Proteins, RNAs, and genes were identified using their official symbols from the HUGO Gene Nomenclature Committee ( [HGNC](https://www.genenames.org) ), allowing the MINERVA platform, which facilitates visualization and navigation of the map (see details below), to automatically add further annotations such as Ensembl, Entrez Gene, RefSeq, and UniProt IDs. If HGNC could not recognize certain symbols or terms, additional annotations were manually assigned. Complexes were annotated, when available, to Gene Ontology (GO) cellular component terms and their corresponding IDs, while metabolites were identified by their [ChEBI names](https://www.ebi.ac.uk/chebi) and IDs. Phenotypes were identified using GO biological process terms and their IDs ( see the [GO documentation](http://geneontology.org/docs/ontology-documentation) ) for biological processes, or Medical Subject Headings (MeSH) terms and their IDs (https://www.ncbi.nlm.nih.gov/mesh) for disease-related elements. Annotations not automatically retrieved by MINERVA were added manually via the MIRIAM section of CellDesigner. References for interactions, including PubMed IDs of the source papers, were annotated using the relation “bqbiol: isDescribedby”, while other annotations were added using the “bqmodel:isEncodeby”.

## Map availability and visualisation in the MINERVA platform
The ISD map is accessible as an online interactive map via the Molecular Interaction Networks Visualization (MINERVA) platform (https://imi-biomap.elixir-luxembourg.org/minerva/). MINERVA is a standalone web server designed for the visual exploration, analysis, and management of molecular networks encoded in systems biology formats such as CellDesigner, SBML, and SBGN. It offers automated content annotation and verification, along with features like overlaying experimental data (e.g., transcriptomics, gene variant data) on the visualized networks. For more information on MINERVA's functionalities, please refer to the documentation at https://minerva.pages.uni.lu/doc/. We also provide a quick [guide](isd_guide.md) on how to navigate and use the ISD map.

## Integration, visualisation and analysis of omics data
### Integration of Open Targets Genetics data to the map
For the analysis of possible mechanistic effects of gene variants on downstream molecular processes in ISD map, we collect AD- and PsO-associated genes, i.e., genes harbouring variants, such as single nucleotide polymorphisms (SNPs), associated with AD and PsO, respectively, from the [Open Targets Genetics database](https://doi.org/10.1093/nar/gkaa840). We used the Experimental Factor Ontology (EFO) identifiers of AD (EFO_0000274) and PsO (EFO_0000676) as queries in the Open Targets Platform’s home page (https://platform.opentargets.org). Next, from the results page we downloaded only the dataset “Genetic Association”, with no other selected option, as a TSV file. We prepared MINERVA-compatible files for the creation of visual overlays of these variants in ISD map and uploaded them to the MINERVA platform for the creation of the following overlays: the “Omics: Genes harboring AD-associated SNPs from Open Targets Genetics database” in the AD map and the “Omics: Genes harboring PsO-associated SNPs from Open Targets Genetics database” in the PsO map.

### Integration of proteomics and transcriptomics data to the map  
For the analysis of omics data integration with the map, we used two datasets: one proteomic dataset related to AD ([He et al 2020](https://doi.org/10.3389/fimmu.2020.01768)) and a transcriptomic dataset related to PsO ([Tian et al 2012](https://doi.org/10.1371/journal.pone.0044274)). 
The AD-related proteomic data was collected from the study by [He et al 2020](https://doi.org/10.3389/fimmu.2020.01768); in this study, inflammatory proteome expression profiles (based on a panel of pre-selected 353 inflammatory proteins) were measured via Olink proteomic assay in AD lesional and non-lesional skin samples taken from patients before and after treatment with dupilumab. For our integration, we considered only differentially expressed proteins (DEPs) calculated by comparing expression profiles extracted from lesional skin samples of patients before and after dupilumab exposure. DEPs with FDR < 0.05 and fold-change (FCH) > 1.5 were collected from the paper’s [Supplementary Table E2](https://www.frontiersin.org/articles/10.3389/fimmu.2020.01768/full#supplementary-material), specifically from columns A (HGNC symbol of the proteins), H (“FCH LS Post-Rx versus Pre-Rx”) and J (“FDR LS Post-Rx versus Pre-Rx”). We then normalized the FCH of DEPs to the [-1,1] range. Finally, we prepared the file for the creation of a visual overlay of these DEPs in ISD map in the MINERVA platform: the overlay “Omics: Differentially expressed proteins from dupilumab vs untreated lesional skin He et al (2020)”.
The PsO-related transcriptomic data was collected from the study by [Tian et al 2012](https://doi.org/10.1371/journal.pone.0044274); in this study, differentially expressed genes (DEGs) were defined via a meta-analysis of five transcriptomic studies ([GSE6710](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE6710), [GSE11903](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE11903), [GSE14905](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE14905), [GSE13355](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE13355) and [Suárez-Fariñas et al 2012](https://www.jidonline.org/article/S0022-202X(15)35503-2/fulltext) comparing lesional and non-lesional skin samples (the MAD-5 transcriptome). For our integration, we considered only DEGs with FDR < 0.05 and fold-change (FCH) > 2 extracted from the [Supplementary Table S2](https://doi.org/10.1371/journal.pone.0044274.s003). We then normalized the FCH of DEGs to the [-1,1] range and prepared the file for the creation of a visual overlay of these DEGs in ISD map in the MINERVA platform: the overlay “Omics: Differentially expressed genes from the meta-analysis lesional vs non-lesional PsO skin (MAD5 transcriptome)”.

### Functional enrichment analysis in MINERVA  
To check if some of intra- or intercellular activities in both ISD map were enriched in AD- and PsO-related omics-prioritised genes, we used the MINERVA’s GSEA plugin as described in details [here:](https://minerva.pages.uni.lu/doc/plugins/gsea-plugin/). In brief, the MINERVA’s GSEA plugin considers as the background gene list all genes present in the map; as the pathway database source, the plugin considers the list of annotated pathways in the own map. The statistical test used is the hypergeometric test and the obtained p-values are adjusted for multiple comparisons by using the Bonferroni test.  Enriched pathways were those with adjusted p-value < 0.05. 

## Boolean network and simulation  
We used Probabilistic Boolean Modelling (PBM) to simulate the effects of IFNG on sensory perception of itching after treatment with dupilumab, i.e., inhibition of IL4R. For this purpose, we first converted the individual KC map in the AD map into a Boolean network (BN) in an automated fashion using [CaSQ tool](https://casq.readthedocs.io/en/stable/). Then we considered the following pathways in the converted BN for our simulations: i) IL4R -> TSLP -> sensory perception of itching, (ii) IL4R -> KLK5 -> sensory perception of itching, (iii) IL4R -> KLK7 -> sensory perception of itching, (iv) IFNG -> TSLP -> sensory perception of itching, (v) IFNG -| KLK5 -> sensory perception of itching and (vi) IFNG -| KLK7 -> sensory perception of itching. The PBM approach uses a series of random walks to determine the probability of components within the model ([Hemedan et al 2022](https://doi.org/10.1016/j.csbj.2022.06.035)). This approach integrates qualities of both discrete and continuous Markov processes within a Monte Carlo framework ([Hemedan et al 2023](https://doi.org/10.3389/fbinf.2023.1189723)). To establish a foundational baseline for our simulations, we parameterized initial state probabilities of ON/OFF. This step is critical as it sets the starting point for the model, reflecting the pre-simulation status of molecular interactions. The equation for updating the state probabilities is given by:  
![Alt text](projects/images/formula.png)  
where Pt+1(s) is the probability of state s at the next time point and P(s′∣s) is the transition probability from a previous state s′ to the current state s. 
