---
layout: page
title: ISD Guide
permalink: /isdguide/
---


# A quick user guide to navigate and explore the ISD map  

## Accessing the maps

The main entry point to the ISD map is available at https://imi-biomap.elixir-luxembourg.org/. This is a side-by-side simplified comparison of the contents of atopic dermatitis (AD) and psoriasis (PsO) maps. To access the maps, follow the steps below.




![step1](https://github.com/user-attachments/assets/f5cab09b-faf6-4d10-a367-d3928dce6c2d)



To access the individual, i.e., AD or PsO, intercellular communication maps, please click on the button with disease name above each part of the diagram.  

 ![step2](https://github.com/user-attachments/assets/3187aaaf-da58-41f7-97dd-c51387d516e6)


Clicking on the “ATOPIC DERMATITIS” button, you will be directed to the AD map. 

![step3](https://github.com/user-attachments/assets/9d284197-04e1-47b4-a53a-b3a1498a4b8b)


## Exploring the maps
You can explore the map by navigating it similarly as you do in Google Maps. Use the buttons on the lower right corner to zoom in, zoom out and center the map content.
 
![step4](https://github.com/user-attachments/assets/cc0e362b-a196-46b6-974b-640eb1ac9e63)


To access information about map elements - cell types, phenotypes, proteins, genes, interactions - click on the element, e.g. CCL2, which becomes marked with a blue anchor, and check the left panel, which contains annotations referring to the element.

 ![step5](https://github.com/user-attachments/assets/053df113-5608-47ac-b586-32695495e260)


An arrow connecting elements on the map (a reaction) represents an interaction between them. By clicking on it, it becomes blue and the left panel displays references to publications supporting this reaction, and elements linked by the interaction.

 ![step6](https://github.com/user-attachments/assets/baf2be09-f6c7-4b5a-9076-63032f0b2c05)


## Search
You can search for elements using the search panel (as shown below). After clicking on  the magnifier or hitting “Enter”, the searched element, e.g. IGHE, will be marked with blue anchors in the map. Select “Perfect match” to focus search results.  

![step7](https://github.com/user-attachments/assets/f838d8cd-bb85-4192-8d24-28755460ffc9)

 

You can access submaps containing intracellular pathways in key cell types in the intercellular communication map. This can be done by clicking the “Submaps” button, then selecting the submap of interest among the listed ones in the panel appearing in the left side, and finally clicking “ >” in the same row of the selected submap, e.g., Th1.	

 ![step8](https://github.com/user-attachments/assets/1ad57923-a2b3-45ae-8b97-011f89d256d2)


After clicking “>” in the same row of the submap “Th1”, as shown above, you are directed to the submap containing AD-related intracellular pathways in Th1 cells. If you would like to have a broader view of the submap, close the “Submaps” panel by clicking on “X”. To check if you are in the desired submap, look for the tab colored in light blue.

![step9](https://github.com/user-attachments/assets/27168a2c-a8ca-46db-a438-485210d22e12)

 
**IMPORTANT**: you can perform exploration and search activities in submaps.

## Data visualisation

Click the “Overlays” button to visualise i) genes harboring AD- or PsO-specific SNPs , ii) differentially expressed genes and proteins with their normalized log fold change (LFC) values and iii) AD- and PsO-linked biological processes discussed in the paper. The left panel will display the above-mentioned list. Click “View” to visualise an overlay of interest, e.g., the overlay “Omics: Differentially expressed proteins from dupilumab vs untreated lesional skin He et al (2020)”.

![step10](https://github.com/user-attachments/assets/46f2fd28-92d7-45d9-b457-837251327752)


After clicking “View”, the elements present in selected overlays containing datasets with differential expressed genes or proteins along with their normalized LFC values – in this case, for example, the overlay “Omics: Differentially expressed proteins from dupilumab vs untreated lesional skin He et al (2020)”, are highlighted in a gradient color according to their normalized LFC values as shown in the horizontal bar at the bottom of the panel “Overalys”. If you want to remove the highlights, click “Hide”.

 ![step11](https://github.com/user-attachments/assets/c95687ab-408b-4a56-8989-53f512340bf1)


If the selected overlay contains a dataset with gene symbols along with a fixed color, e.g., orange, as it happens for the overlay “Omics: Genes harboring AD-associated SNPs from Open Targets Genetics database”, you see, after clicking “View”, the corresponding genes highlighted in orange in this map. The color is shown in the horizontal bar at the bottom of “Overalys”. If you want to remove the highlights, click “Hide”.

![step12](https://github.com/user-attachments/assets/02627fea-4146-4378-987c-fdbcac221df1)

