---
title: DMCM2024 Programme
layout: default
permalink: /DMCM2024/HenningHermjakob/
---

<img src="/images/places/Belval08.jpg"/>

[[back to the DMCM'24 Programme]](https://disease-maps.org/DMCM2024/programme/)

<table>
  <tr>
    <td style="width: 140px;">
      <img src="/images/teamhq/HenningHermjakob.jpg" width="135"/></td>
    <td> 
      <a href="https://www.ebi.ac.uk/people/person/henning-hermjakob/" target="_blank"><b>Henning Hermjakob</b></a>, Head of Molecular Systems Services, European Bioinformatics Institute (EMBL-EBI), Cambridge, UK
    </td>
  </tr> 
</table>

### Network resources for FAIR modeling

The Molecular Networks team at the European Bioinformatics Institute provides resources that focus on the interactions and functional connectivity of biomolecules, spanning protein, oligonucleotide and small molecule components. These data reflect the coordinated self-organization and assembly of multi-component, regulated and emergent complex biological functions, building on reference resources such as ChEBI, UniProt, Ensembl, and the Complex Portal. The resources provide network data at increasing levels of abstraction, from primary molecular interaction data in IntAct via aggregated and curated pathway data in Reactome to mathematical models of biological systems in BioModels. 
Since its inception, the IntAct molecular interaction database (Del Toro et al. 2022) has emphasized a detailed, “deep” curation model, abstracting not only interaction partners, but also interaction details like experimental context, binding sites, PTMs, and mutations from the scientific literature. This detailed curation has allowed us to publish, among others, a key reference dataset containing 28,000 annotations fully describing the effect of individual point sequence changes on physical protein interactions, abstracted from the annotation of all partners in the International Molecular Exchange consortium (IMEx) partners over 14 years.
The Reactome database of curated human pathways (Milacic et al, 2024) currently covers 11,392 proteins in 15,046 reactions supported by 37,933 literature references (Release 87, December 2023). Building on our successful collaboration with the DiseaseMaps community (Ostaszewski et al, 2021), we are increasing our efforts in pandemic preparedness, among others through the recent release of the Respiratory Syncytial Virus (RSV) Infection Pathway. We are now also providing “early access” to pathways still awaiting external review, to extend the available data, and to invite community collaboration as in the Covid-19 Disease Maps project. Pathway analysis capabilities range from simple gene enrichment to quantitative multi-omics analysis with ReactomeGSA (Griss et al. 2020). Analysis can be submitted through an R interface, and results are presented interactively in a visually attractive genome-wide results overview based on Voronoi maps. ReactomeGSA now includes a function that enables users to directly search for public datasets in Expression Atlas (Moreno et al., 2022) and GREIN (Mahi et al., 2019) from within ReactomeGSA, that can then be loaded into a ReactomeGSA analysis. These new features are available through a completely re-developed web interface as well as the ReactomeGSA Bioconductor R package.
The BioModels database provides an open resource of high quality, validated systems biology models in the community standard SBML representation, an essential service, as research shows that around 50% of published systems biology models are not directly reproducible from the associated publication (Tiwari et al. 2021). Building on these results, we will discuss recent approaches and challenges in reproducing machine learning based models. 
All resources are open source, open data, distributed under permissive licenses (CC0, CC-BY).


<img src="/pages/events/DMCM2024/fig1.jpg"/>
<!--<img src="/pages/events/DMCM2024/fig1.jpg" width="600"/>-->

