---
title: DMCM2024 Abstract
layout: default
permalink: /DMCM2024/Abstracts/S25
---

**Title:** \
Network footprint analysis: an efficient tool to facilitate precision medicine

**Authors:** \
Dezso Modos, Deema Alassaf and Tamas Korcsmaros

**Abstract:** \
Recently, many signalling and regulatory network resources and biological pathway databases were developed. One of the applications of these network databases is network footprint analysis. In network footprint analysis, one connects the input (perturbagent) with the response (perturbation) in a signalling network. Most of the time, the perturbagen is some kind of compound, but it can be a virus or genetic change as well. The output response can be ideally captured with phosphoproteomics, however more often transcriptomics is used due to the easier accessibility and coverage to this omics layer.
Previously, we used transcriptomic network footprint analysis to uncover pathomechanism of SARS-CoV-2 infection in the gut from single-cell data of infected intestinal organoids. Similar methodology was used to better understand the mechanisms of toxicity of drug induced liver injury compounds. Using network footprints from patient specific data, we uncovered patient specific pathogenic pathways. We have analysed such network footprints in ulcerative colitis and Crohn's disease, the two major types of inflammatory bowel disease. Drug specific network footprints from drug targets and transcriptional changes in response to drug treatment creates drug specific network footprints. Signature matching is a tool to compare two perturbagents such as a disease-associated genetic change and a drug at the transcriptome level. The outcome of signature matching could be (1) the signatures match indicating a similar phenotype (2) anticorrelate alleviating each otherâ€™s phenotype predicting that the drug treats the disease (3) the signatures do not match and are not anti correlated meaning there is no effect. Network footprints have the advantage compared to transcriptomic data that biological networks dampen the transcriptomic noise and also suggest a mechanistic understanding of the process whether that is the pathogenesis of the disease or mechanism of action of a drug. Network footprints can replace the transcriptome in signature matching expecting a better outcome. Network footprint based signature matching of drugs and individual patients could predict the right drug to the right patient.
Network footprint based signature matching could be specifically useful to facilitate precision medicine for chronic and complex diseases, such as inflammatory bowel disease where the current success rate of the best drugs is only around 30%.

